src/InteractiveWindow/Version.props

```diff
@@ -1,5 +1,5 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <RoslynSemanticVersion>1.2.0</RoslynSemanticVersion>
+    <RoslynSemanticVersion>2.0.0</RoslynSemanticVersion>
```

Comment:
@KevinH-MS @amcasey No, it's not possible. We use these version numbers to version the NuGet packages. We can't generate NuGet packages with the same version that are different packages.